src/workers/parser/getSymbols.js

```diff
@@ -199,7 +200,12 @@ function extractSymbol(path: SimplePath, symbols) {
       return;
     }
 
-    if (t.isProperty(path.parent)) {
+    if (
+      path.parent &&
+      t.isProperty(path.parent) &&
+      !t.isAssignmentPattern(path.getAncestor(3)) &&
+      !(t.isCallExpression(path.getAncestor(3)) && keyIsValue(path.parent))
```

Comment:
maybe keyIsValue should be isObjectShorthand

Reply:
I knew their had to be a better way of saying this haha.